README.md

```diff
@@ -17,13 +17,19 @@ Install with:
 npm install --save-dev @mrgrain/jsii-struct-builder
 ```
 
+Then, place a `new ProjenStruct` in your `.projenrc.js` file, passing your
+[TypeScript project](https://projen.io/typescript.html) as the first parameter. See the sections below for more usage
+details.
```

Comment:
```suggestion
Then add a `new ProjenStruct` in your `.projenrc.ts` file, passing a [TypeScript project](https://projen.io/typescript.html) as the first parameter.
See the sections below for more usage details.
```

Reply:
Yes!

https://projen.io/typescript.html#migrating-projenrcjs-to-projenrcts

New projects are setup with it by default! Looks like your projen version is out of date 😬 
```console
$ npx projen@latest new awscdk-construct
Need to install the following packages:
  projen@0.71.91
Ok to proceed? (y) y
👾 Project definition file was created at /Users/kornherm/code/scratchpad/pts/.projenrc.ts
```